tools/serverpod_cli/test/database/create_definition/database_action_definition_test.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:serverpod_cli/src/database/create_definition.dart';
+import 'package:serverpod_cli/src/test_util/builders/class_definition_builder.dart';
+import 'package:serverpod_cli/src/test_util/builders/foreign_relation_definition_builder.dart';
+import 'package:serverpod_cli/src/test_util/builders/serializable_entity_field_definition_builder.dart';
+import 'package:serverpod_service_client/serverpod_service_client.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test(
+      'Given a class definition with a table, then generate a table with that name.',
+      () {
+    var field = FieldDefinitionBuilder().withPrimaryKey().build();
+    var entity = ClassDefinitionBuilder()
+        .withTableName('example')
+        .withField(field)
+        .build();
+
+    var databaseDefinition = createDatabaseDefinitionFromEntities([entity]);
+
+    expect(databaseDefinition.tables, hasLength(1));
+    expect(databaseDefinition.tables.first.name, 'example');
+  });
+  group(
+      'Given a class definition with a foreign relation with onDelete and onUpdate set to "SetNull"',
+      () {
+    var relation = ForeignRelationDefinitionBuilder()
+        .withParentTable('example')
+        .withReferenceFieldName('id')
+        .withOnDelete(ForeignKeyAction.setNull)
+        .withOnUpdate(ForeignKeyAction.setNull)
+        .build();
+
+    var field = FieldDefinitionBuilder()
+        .withName('parentId')
+        .withIdType(true)
+        .withRelation(relation)
+        .build();
+
+    var entity = ClassDefinitionBuilder()
+        .withTableName('example')
+        .withField(field)
+        .build();
+
+    var databaseDefinition = createDatabaseDefinitionFromEntities([entity]);
+
+    var failedPrecondition = databaseDefinition.tables.isEmpty;
```

Comment:
Suggestion:
```suggestion
    var tablesDoNotExist = databaseDefinition.tables.isEmpty;
```

Reply:
Explain what the precondition so that it makes sense in the "skip" place. Should also give a reason for the skip so that it can be understood when running all the test.

Might apply to more places.